model-optimizer/extensions/front/tf/sparse_to_dense_replacer.py

```diff
@@ -21,8 +21,8 @@ class SparseToDenseReplacer(FrontReplacementOp):
     enabled = True
 
```

Comment:
please don't move this transformation into `front/tf`. Only correct run_after transformations as you did

Reply:
I did as you described and got a lot of errors while non TF model conversions. Looks like we have deep problem inside our class registration.

Reply:
Also it is TF specific transformation too, why we cant move it to `front/tf` folder?

Reply:
I am fine with your proposal but let us check W&D model for which this transformation is targeted.

Reply:
Ok, I will check it.

Reply:
According to e2e results tests with this and another affected model are passed.